src/test/java/com/arpnetworking/logback/serialization/steno/StenoSerializationHelperTest.java

```diff
@@ -32,19 +35,30 @@
  */
 public class StenoSerializationHelperTest {
 
-    @Before
-    public void setUp() {
-        _objectMapper = new ObjectMapper();
-        _encoder = new StenoEncoder();
-        _objectMapper.setAnnotationIntrospector(new StenoAnnotationIntrospector(_objectMapper));
+    @Test
+    public void testNonThrowableProxy() throws IOException {
+        final ObjectMapper objectMapper = new ObjectMapper();
+        final StringWriter jsonWriter = new StringWriter();
+        final JsonGenerator jsonGenerator = objectMapper.getFactory().createGenerator(jsonWriter);
+        jsonGenerator.writeStartObject();
+        objectMapper.setAnnotationIntrospector(new StenoAnnotationIntrospector(objectMapper));
+        StenoSerializationHelper.serializeThrowable(
+                new AlternateIThrowableProxy(),
+                jsonGenerator,
+                objectMapper);
+        jsonGenerator.writeEndObject();
+        jsonGenerator.close();
+        Assert.assertEquals(
```

Comment:
I want to see non null fields on the AlternateIThrowableProxy. That way we can be sure that if logback decides to change the underlying type, we will still get a message and stack trace. 

As things are now, the nulls in your expected output could be masking an error in the code. While that seems unlikely, it would be nice to have a test that proves things work.

Does that make sense?